actionview/test/template/text_helper_test.rb

```diff
@@ -68,6 +68,8 @@ def test_simple_format_should_not_sanitize_input_when_sanitize_option_is_false
 
   def test_simple_format_with_custom_wrapper
     assert_equal "<div></div>", simple_format(nil, {}, { wrapper_tag: "div" })
+    assert_equal "<p></p>", simple_format(nil, {}, { wrapper_tag: "" })
+    assert_equal "<p></p>", simple_format(nil, {}, { wrapper_tag: nil })
```

Comment:
So 👍 to have `nil` just use the default `p` tag.

However I'm not very fan of `""` being handled as `nil`, as it's likely to be an error. I'd rather raise an argument error on `blank?`.

Reply:
I completely understand your concern about treating an empty string `""` as `nil`, as it can potentially hide errors or unexpected behavior in the code.

Reply:
Perhaps we should validate the tag only contain allowed characters? That would perhaps even prevent some XSS.

A quick search show that allowed characters are:

> alphanumeric characters ( [A-z|a-z|0-9] ), and a period ( . ), an underscore ( _ ), or a dash ( - ).

But I wonder if there's more to it. @flavorjones as an HTML expert (😉), what you you think?

Reply:
Yeah that would be great :)

Reply:
WDYT for creating another PR for adding the validation for content_tag. For this PR we can only scope for `nil` case?

Reply:
Fine by me. The content tag validation will need a bit more discussion anyway, so might as well just fix the nil case here.

Reply:
Ok. I'm removing other changes

Reply:
Updated the changes